docs/spec/annotations.rst

```diff
@@ -114,6 +114,16 @@ be evaluated.
 Moreover, the expression should be parseable as a valid type hint, i.e.,
 it is constrained by the rules from the section on :ref:`valid-types`.
 
+If a triple quote is used, the string should be parsed as though it is
+implicitly surrounded by parentheses. This allows newline characters to be 
+used within the string literal.
+
+  value: """
+      int |
+      str |
+      list[Any]
+    """
```

Comment:
Can you double-check that this renders correctly as a text block? I'm no Sphinx expert, but IIRC text blocks are normally introduced by a double colon, as in the example a few lines down.

Reply:
Oh, good point. I'm not very experienced with Sphinx notation, so I may have gotten this wrong. Perhaps someone more knowledgeable could help here.

Reply:
```suggestion
used within the string literal::

    value: """
        int |
        str |
        list[Any]
    """
```